Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/EditorMaterialComponentSlot.cpp

```diff
@@ -241,9 +227,11 @@ namespace AZ
                 OnPropertyChanged();
             };
 
-            if (m_materialAsset.GetId().IsValid())
+            Data::Asset<RPI::MaterialAsset> assetToEdit = m_materialAsset.GetId().IsValid() ? m_materialAsset : m_defaultMaterialAsset;
+
+            if (assetToEdit.GetId().IsValid())
             {
-                if (EditorMaterialComponentInspector::OpenInspectorDialog(GetLabel(), m_materialAsset.GetId(), m_propertyOverrides, applyPropertyChangedCallback))
+                if (EditorMaterialComponentInspector::OpenInspectorDialog(GetLabel(), assetToEdit.GetId(), m_propertyOverrides, applyPropertyChangedCallback))
```

Comment:
After discussing offline, this should be reverted until we get feedback on how to handle editing default material overrides

Reply:
I was expecting we'll do it this way eventually so was going to leave it, but okay it's easy enough to revert.